packages/graph/hash_graph/lib/graph/src/store/mod.rs

```diff
@@ -242,14 +246,11 @@ pub trait PropertyTypeStore {
     /// # Errors
```

Comment:
Comment on line 244 shouldn't have `identifier`. I think this is the case for all the `get_*_type` methods as well as `get_entity`

Reply:
Applied in 4dfc8ec7ba2081be1564694186602ff6cc00b748